pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -173,46 +173,46 @@ def nodes_visited_for_all_seq_ref(len_seq_list, distance,
+     * ISAX distribution of sequences to index
+     * and towards the first root node
 
     .. WARNING::
-        Dans cette version, les data_ts sont obligatoires pour définir à l'avance les futurs breakpoints
+        In this version, the data_ts are mandatory to define in advance the future breakpoints
 
-    :param int size_word: le nombre de discrétisation SAX pour chaque séquence
-    :param int threshold: la capacité maximale des nœuds feuilles de l'arbre
-    :param numpy.ndarray data_ts: array de séquences à insérer
-    :param int base_cardinality: la plus petite cardinalité pour l'encodage iSAX
-    :param int max_card_alphabet: si self.boolean_card_max == True, cardinalite max pour l'encodage iSAX
+    :param int size_word: The number of Sax discretization for each sequence
+    :param int threshold: The maximum capacity of the nodes of the tree
+    :param numpy.ndarray data_ts: Sequence array to be inserted
+    :param int base_cardinality: The smallest cardinality for encoding iSAX
+    :param int max_card_alphabet: if self.boolean_card_max == True, Max cardinalite for encoding iSAX
 
 
-    :ivar int size_word: Nombre de lettres contenues dans les mots SAX indexés dans l'arbre
-    :ivar int threshold: Seuil avant la séparation d'une feuille en deux nœuds feuilles
+    :ivar int size_word: Number of letters contained in the SAX words indexed in the tree
+    :ivar int threshold: Threshold before the separation of a sheet into two leaf nodes
     """
 
     def __init__(self, size_word, threshold, data_ts, base_cardinality=2, max_card_alphabet=128,
                  boolean_card_max=True):
         """
-        Fonction d'initialisation de la classe TreeISAX
+        Class initialization function TreeISAX
 
-        :returns: un arbre iSAX
+        :returns: a tree iSAX
         :rtype: TreeISAX
         """
         
-        # Nombre de lettre contenues dans les mots SAX indexés dans l'arbre
+        # Number of letters contained in the SAX words indexed in the tree
         self.size_word = size_word
-        # Seuil avant la séparation d'une feuille en deux nœuds feuilles
+        # Threshold before the separation of a sheet into two leaf nodes
         self.threshold = threshold
-        # cardinalité de chacune des lettres au niveau 1 de l'arbre
-        # n'est pas très utile...
+        # Cardinality of each letter at level 1 of the tree
+        # is not very useful ...
         self._base_cardinality = base_cardinality
-        # cardinalité max courante
+        # Current Max Cardinality
         self.bigger_current_cardinality = base_cardinality
-        # Si vrai, défini une cardinalité maximale pour l'alphabet de l'arbre
+        # If true, defined maximum cardinality for the alphabet of the tree
         self.boolean_card_max = boolean_card_max
         self.max_card_alphabet = max_card_alphabet
 
-        # mean, variance des sequences data_ts
+        # mean, variance of data_ts sequences
```

Comment:
(my mistake) not "variance" but "écart-type" so -> standard deviation

Reply:
```suggestion
        # mean, standard deviation of data_ts sequences
```